test/sql/aggregate/qualify/test_qualify_view.test

```diff
@@ -37,21 +37,29 @@ ORDER BY ALL
 22	11
 
 # drop the table the view is based on
-statement ok
+statement error
```

Comment:
This is probably not the behavior we want, so we should allow `test` to be dropped if the only dependencies are views.
We do need to keep them as dependencies for the sake of exporting, the table has to be written before the view is written